core/templates/dev/head/pages/exploration_player/exploration_footer_directive.html

```diff
@@ -0,0 +1,134 @@
+<script type="text/ng-template" id="components/explorationFooter">
+  <div class="footer navbar-fixed-bottom oppia-exploration-footer">
+    <div class="row">
+      <div class="col-sm-6">
+        <ul class="author-profile">
+          <li>
+            <img ng-src="<[getStaticImageUrl('/general/apple.svg')]>" class="exploration-footer-img">
+          </li>
+          <li class="hover-link"
+              ng-mouseenter="hovering=true"
+              ng-mouseleave="hovering=false">
+            <div class="dropup" ng-class="{'open':hovering}">
+              <a id="dropdownMenu2" class="dropdown-toggle" ng-class="{'hovered-text':hovering}" data-toggle="dropdown"
+                 ng-if="contributorNames.length > 0">
+                <h4 translate="I18N_FOOTER_AUTHOR_PROFILE" class="author-profile-text"></h4>
+              </a>
+              <ul class="author-profile-dropdown-menu dropdown-menu" aria-labelledby="dropdownMenu2" role="menu">
+                <li ng-repeat="name in contributorNames | limitTo: 5" style=>
+                  <[name]>
+                </li>
+              </ul>
+            </div>
+          </li>
+          <li></li>
+        </ul>
+      </div>
+      <div class="col-sm-6">
+        <div class="pull-right">
+          <ul class="author-profile">
+            <li>
+              <h4 translate="I18N_PLAYER_SHARE_THIS_EXPLORATION" class="share-exploration-footer"></h4>
+            </li>
+            <li>
+              <sharing-links ng-if="explorationStatus !== 'private'"
+                             ng-click="$event.stopPropagation()"
+                             twitter-text="getTwitterText()"
+                             exploration-id="explorationId"
+                             layout-type="row" 
+                             class="oppia-exploration-footer-sharing-links">
+              </sharing-links>
+            </li>
+          </ul>
+        </div>
+      </div>
+    </div>
+  </div>
+</script>
+
+<style>
+
+.oppia-exploration-footer {
+  background-color: #094142;
+  color: #FFF;
+  font-family: "Capriola", "Roboto", Arial, sans-serif;
+  font-size: 13px;
+  height: 42px;
+  line-height: 10px;
+  margin-top: 30px;
+  width: 100%;
+}
+
+.exploration-footer-img {
```

Comment:
Prefix every selector with .oppia-exploration-footer, so that the styles in this file apply only to the footer and don't proliferate all over the application.